all.js

```diff
@@ -359,7 +391,7 @@ function localRetrieve(url, options, cb) {
  * @returns {string}
  */
 function getToolsFolder() {
-  var folder = process.env[(process.platform == 'win32') ? 'USERPROFILE' : 'HOME'] + '/vsc-iot-tools';
+  var folder = process.env[(process.platform == 'win32') ? 'USERPROFILE' : 'HOME'] + '/.iot-hub-getting-started';
```

Comment:
yes, current way definitely works cross-platform, but your suggestiong is definitely better. currently path looks ugly when printed out on windows.